api/nakadi-event-bus-api.yaml

```diff
@@ -416,28 +416,35 @@ paths:
       security:
         - oauth2: ['nakadi_event_stream.write']
       description: |
-        Publishes one `Event` of this `EventType`. (**Note** *Format and response semantics of publishing of
-        batches of Events will be added in the short term.*)
+        Publishes a batch `Event`s of this `EventType`. All items must be of the same EventType.
 
         Reception of Events will always respect the configuration of its `EventType` w.r.t. validation,
         enrichment and partitioning. The steps performed on reception of incoming message are:
 
         1. Every validation rule specified in the `EventType` will be checked in order against the incoming
         Events. Validation rules are evaluated in the order they are defined and the Event is **rejected**
-        in the first case of failure with a status of 400 Bad request. If the offending validation rule
-        provides information about the violation it will be included in the Problem response.
+        in the first case of failure. If the offending validation rule
+        provides information about the violation it will be included in the `BatchItemResponse`.
         If the `EventType` defines schema validation it will be performed at this moment.
 
         1. Once the validation succeeded, the content of the Event is updated according to the enrichment
         rules in the order the rules are defined in the `EventType`.
         No preexisting value might be changed (even if added by an enrichment rule). Violations on this will
-        force the immediate **rejection** of the Event with a 409 Conflict status. The invalid overwrite
-        attempt will be included in the response's Problem object.
+        force the immediate **rejection** of the Event. The invalid overwrite
```

Comment:
- the status is actually changed only for that event (if event i is invalid, this does not make even i+1 invalid).
- the impact on the streaming is explained in a paragraph below.